mix/qml/js/ProjectModel.js

```diff
@@ -553,10 +588,10 @@ function normalizeAddress(addr)
 	var i = 0;
 	for (var k in addr)
 	{
-		if (addr[k] !== "0")
+		if (addr[k] !== "0" || addr.length === 40)
```

Comment:
This check can be placed outside of the loop. I guess in this function you can just return 40 rightmost characters